_assets/docs/DEVELOPMENT_GUIDELINES.md

```diff
@@ -0,0 +1,11 @@
+This document aims to group all general guidelines to have in mind when you work with status-go.
+
+
+### /geth/ path
```

Comment:
does it make sense to create this guideline? maybe just move all code from geth namespace?

Reply:
👍  i can create an issue instead